lib/theme/themes.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:flutter/material.dart';
+
+class ChatAppThemes {
```

Comment:
a może factory?

Reply:
właściwie to nie wiem ile byśmy chcieli mieć tych motywów, bo nwm czy jest sens robić więcej niż dwa typu jasny/ciemny

Reply:
niezależnie ile by nie było - tak będzie łatwo dodać nowy